Gemfile

```diff
@@ -60,6 +60,8 @@ gem 'idn-ruby', require: 'idn'
 gem 'kaminari', '~> 1.2'
 gem 'link_header', '~> 0.0'
 gem 'mime-types', '~> 3.4.1', require: 'mime/types/columnar'
+gem 'net-imap' # Remove me after mail upgraded to 2.8.0
+gem 'net-pop' # Remove me after mail upgraded to 2.8.0
```

Comment:
Mail *has* been updated to 2.8.0

Reply:
Thanks, I didn't catch that.  It was not upgraded at the time I'd made the commits. 🙇 

I've removed the commit that added this.